pyro/distributions/transforms/affine_autoregressive.py

```diff
@@ -5,10 +5,13 @@
 import torch.nn as nn
 from torch.distributions import constraints
 
+from pyro.distributions.conditional import ConditionalTransformModule
 from pyro.distributions.torch_transform import TransformModule
 from pyro.distributions.util import copy_docs_from
 from pyro.distributions.transforms.utils import clamp_preserve_gradients
-from pyro.nn import AutoRegressiveNN
+from pyro.nn import ConditionalAutoRegressiveNN, AutoRegressiveNN
+
+from functools import partial
```

Comment:
nit: can you normalize import order using `isort`?
```sh
pip install isort
isort -iy  pyro/distributions/transforms
```

Reply:
Done for this and other transforms!